packages/block-editor/src/components/block-list/index.js

```diff
@@ -183,9 +187,20 @@ function Items( {
 					getBlockEditingMode,
 					__unstableGetEditorMode,
 				} = select( blockEditorStore );
+
+				const _order = getBlockOrder( rootClientId );
+
+				if ( getSettings().__unstableIsPreviewMode ) {
+					return {
+						order: _order,
+						selectedBlocks: EMPTY_ARRAY,
+						visibleBlocks: EMPTY_SET,
```

Comment:
this one is interesting because it forces all blocks to be async right? This might have a good impact in terms of performance for the different interactions when previews are rendered.

Reply:
(Idea for later)

I actually wonder if we can't introduce a "priority" number in `priority-queue` at some point where we could say:

 - First render the sync stuff
 - then render async stuff with priority 1
 - and only then (maybe wait a couple async callback before running this list), render async stuff with priority 2

Reply:
They are already async because there is never any selection in previews. What would be priority 2?

Reply:
The early return here is just to prevent the selectors from running, especially `getBlockEditingMode` below.

Reply:
> They are already async because there is never any selection in previews. What would be priority 2?

I mean a way to choose that some components even if async are higher priority than other async components.